ui/plugin-system/src/components/PluginEditor/plugin-editor-api.ts

```diff
@@ -45,16 +48,17 @@ export type UsePluginEditorProps = Pick<PluginEditorProps, 'pluginType' | 'value
  * kind back.
  */
 export function usePluginEditor(props: UsePluginEditorProps) {
-  const { pluginType, value } = props;
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const { pluginType, value, onHideQueryEditorChange = () => {} } = props;
```

Comment:
Could you include a short comment explaining why you have an empty function here and need the eslint-disable? I find that's usually helpful for future us as the code grows.